src/pages/components/pagination.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Pagination
+description: >
+  Pagination separates content into pages and allows users to navigate between those pages.
+package: '@zendeskgarden/react-pagination'
+components:
+  - pagination/src/elements/Pagination/Pagination.tsx
+  - pagination/src/elements/CursorPagination/CursorPagination.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To divide large amounts of data into manageable chunks</li>
+    </ul>
+  </Use>
+</Usage>
+
+## How to use it
+
+### Offset pagination
+
+The basic usage of an offset pagination component.
```

Comment:
I don't think this provides enough of a description, particularly when compared with the definition for cursor based pagination. It'd be helpful to define offset vs. cursor-based.